cmd/kubelet/app/options/osflags_windows.go

```diff
@@ -0,0 +1,32 @@
+// +build windows
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"github.com/spf13/pflag"
+)
+
+var (
+	FlRunService bool
```

Comment:
This should be a subfield of the `KubeletFlags` struct in `options/options.go`. Leave a comment above the field noting that its corresponding flag only gets registered in Windows builds.

Let's also rename it to `WindowsService` here and in kube-proxy so it matches the flag name.

Reply:
Done